### ISSUE TITLE
Address Django 1.11 deprecations

### ISSUE THREAD

**Issue by guillaumep** [santropolroulant/sous-chef#40]
In order to prepare the upgrade to more recent versions of Django we need to address Django's deprecation warnings.

To do so, we can run unit tests using the `-Wa` flag to the python intepreter:

`docker-compose run web python3 -Wa /code/souschef/manage.py test`

We can also check add the `-Wa` flag to the Dockerfile to check the warnings when the web server runs.

**Comment by lauramarec** [comment]
@guillaumep Would that still need to be done?

**Comment by guillaumep** [comment]
@lauramarec Yes, as it will help us upgrade the software libraries. We do not need that immediately though.
